ios/FluentUI/Vnext/Design Token System/FluentUIStyle.generated.swift

```diff
@@ -2737,42 +2737,81 @@ extension FluentUIThemeManagerTheming {
 		open func textCalculatedBackgroundColorsProperty(_ traitCollection: UITraitCollection? = UIScreen.main.traitCollection) -> [UIColor] {
 			if let override = _textCalculatedBackgroundColors { return override }
 			return [
-			UIColor(named: "FluentColors/cyanBlue10", in: FluentUIFramework.resourceBundle, compatibleWith: nil)!, 
```

Comment:
how are we updating our resource bundle?

Reply:
These colors were used by the previous AvatarView, they should be removed when that control goes away. From the YAML file we're only creating hard coded calls to the UIColor API in the generated file.